src/ceph-daemon/ceph-daemon

```diff
@@ -1251,7 +1251,7 @@ def command_enter():
         command = ['bash']
         podman_args += [
             '-it',
-            '--env', 'LANG=C',
+            '-e', 'LANG=C',
```

Comment:
is this part of this PR?

Reply:
not at all. it's not part of my commit
not sure how it got in. I'll fix